rust-backend/README.md

```
@@ -0,0 +1,202 @@
+
+# Or using make commands
+make run
+make build
+make release
+```
+
+The server will start on `http://localhost:5001`
+
+## API Endpoints
+
+### Weather Data
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)